packages/dialog/stories/dialog-wrapper.stories.ts

```diff
@@ -473,6 +473,51 @@ export const wrapperFullscreen = (
     `;
 };
 
+export const wrapperWithHeadline = (
```

Comment:
Story additions are going to bump into this requirement: https://github.com/adobe/spectrum-web-components#keeping-ci-assets-updated